src/components/Dashboard /PartnerData/index.js

```diff
@@ -146,7 +137,28 @@ function PartnerDashboard() {
                     <td data-column="Meraki Link">
                       <div
                         className="create-link"
-                        onClick={() => createMerakiLink(item.id)}
+                        onClick={() => createMerakiLink(item.id, "android")}
```

Comment:
Is the equivalent of this for "web" also done?

Reply:
Sorry. Didn't get you.